server/models/default_local_provider.go

```diff
@@ -191,7 +192,52 @@ func (l *DefaultLocalProvider) HandleUnAuthenticated(w http.ResponseWriter, req
 }
 
 func (l *DefaultLocalProvider) SaveK8sContext(_ string, k8sContext K8sContext) (connections.Connection, error) {
-	return l.MesheryK8sContextPersister.SaveMesheryK8sContext(k8sContext)
+
+	k8sServerID := *k8sContext.KubernetesServerID
+
+	var connID uuid.UUID
+	id, err := K8sContextGenerateID(k8sContext)
+	if err == nil {
+		connID, _ = uuid.FromString(id)
+	}
+
+	_metadata := map[string]string{
```

Comment:
Should this be a Connection, @MUzairS15?

Reply:
It is a connection, the connection specific info is stored here.

Reply:
Because each connection has different schema/properties we store the info in this connection’s  property